server/routes/dashboard/dashboard.test.ts

```diff
@@ -275,8 +275,80 @@ describe('GET dashboard', () => {
         expect(res.text).toContain('To do')
         expect(res.text).toContain('toDo')
         expect(res.text).not.toContain('DRAFT')
-        expect(res.text).toContain('Filter')
-        expect(res.text).not.toContain('Clear')
+        expect(res.text).toContain('Apply filters')
+        expect(res.text).not.toContain('Clear filters')
+        expect(incidentReportingApi.getReports).toHaveBeenCalledWith(expectedParams)
+      })
+  })
+  it('should render expected columns for a user with a single establishment caseload', () => {
+    const mockedReports = [
+      convertBasicReportDates(mockReport({ reportReference: '6543', reportDateAndTime: now })),
+      convertBasicReportDates(mockReport({ reportReference: '6544', reportDateAndTime: now })),
+    ]
+    const pageOfReports = unsortedPageOf(mockedReports)
+    incidentReportingApi.getReports.mockResolvedValueOnce(pageOfReports)
+
+    const users: Record<string, User> = { JOHN_SMITH: { username: 'JOHN_SMITH', name: 'John Smith' } }
+    userService.getUsers.mockResolvedValueOnce(users)
+
+    const expectedParams: Partial<GetReportsParams> = {
+      location: ['MDI'],
+      incidentDateFrom: undefined,
+      incidentDateUntil: undefined,
+      involvingPrisonerNumber: undefined,
+      page: 0,
+      reference: undefined,
+      sort: ['incidentDateAndTime,DESC'],
+      status: undefined,
+      type: undefined,
+    }
+    return request(appWithAllRoutes({ services: { userService }, userSupplier: () => reportingUser }))
+      .get('/reports')
+      .expect('Content-Type', /html/)
+      .expect(res => {
+        expect(res.text).toContain('>Reference number <')
+        expect(res.text).toContain('>Type <')
+        expect(res.text).toContain('>Incident date <')
+        expect(res.text).toContain('>Description<')
+        expect(res.text).toContain('>Reported by <')
+        expect(res.text).not.toContain('>Establishment <')
+        expect(res.text).toContain('>Status <')
```

Comment:
not _that_ big a deal, this test (and the one below) is quite closely tied to how GDS macros render html. i wonder if we could assert the values we expect in the rows  instead of the column titles? e.g. 
```typescript
…
        expect(res.text).toContain('John Smith')
        expect(res.text).not.toContain('Moorland')
…
```